### DISCUSSION TITLE
Privacy method?

### DISCUSSION THREAD

**Discussion by Lockie85** [blakeblackshear/frigate#7414] [Ask A Question] [answered]
Hey,

I've looked through the documentation and I can't find this feature, I wondered if I missed it or if it simply isn't a thing.

I wanted to know if there was a privacy which blocked out or blurred out an area of recorded videos. While I've defined zones for triggering recording, it's still possible for that recording to pick up things I don't want in that recording. As a result I'd like to be able to apply a privacy area which was obscured in some way.

Is this already possible?

Many Thanks

**Comment by NickM-27** [comment, marked as answer]
This feature is already supported using ffmpeg output args. However, you'd be much better off configuring it in your camera or getting cameras that support it.

Adding a privacy mask in frigate will require decoding and re-encoding the recordings which is a lot of work and will also result in worse quality recordings

**Reply by Lockie85** [reply]
Many thanks
